src/exec.js

```diff
@@ -1,14 +1,16 @@
-const childProcess = require('child_process');
+const childProcess = require('child_process'),
+    net = require('net'),
```

Comment:
Is this idiomatic?

Reply:
Probably not. Changed it to be as in the other files.